mesecons_luacontroller/init.lua

```diff
@@ -635,6 +635,7 @@ local function create_environment(pos, mem, event, itbl, send_warning)
 		mem = mem,
 		heat = mesecon.get_heat(pos),
 		heat_max = mesecon.setting("overheat_max", 20),
+		pos = pos,
```

Comment:
The values should be copied to a new table:
```suggestion
		pos = {x = pos.x, y = pos.y, z = pos.z},
```